security/crunch/Portfile

```diff
@@ -20,11 +21,6 @@ homepage        https://sourceforge.net/projects/crunch-wordlist/
 master_sites    sourceforge:crunch-wordlist
 extract.suffix  .tgz
 patchfiles      patch-destroot-fix.diff
-use_configure   no
-universal_variant \
-                no
-build.args-append \
-                CC=${configure.cc}
 destroot.args   PREFIX=${prefix}
```

Comment:
isn't this taken care of by the `makefile` PG as well?

Reply:
I don't think so, if I remove this the file will be put into ${destroot}/usr/

Reply:
It's **very** complicated, and so I can't really be sure, but I _think_ the ```makefile``` PG is trying to set it [here](https://github.com/macports/macports-ports/blob/c2ada0c07b17f080792f457d26a7f9e847943e4e/_resources/port1.0/group/makefile-1.0.tcl#L125).

Reply:
Ken, on that line, the makefile portgroup sets it as an environment variable, but those can be overridden by the makefile, and in the case of this software, that's probably what's happening. IIRC the makefile portgroup way of solving this problem is to use `makefile.override-append PREFIX`.

Reply:
Is there a way to change the override value? In another port i need to set `PREFIX=${destroot}${prefix}`.

Reply:
PG needs to be adjusted so you con figure out how to use it in 10 seconds instead of 10 hours....but there are a lot of yhem in that category I’m afraid.